### DISCUSSION TITLE
OpenAI Assistant API V2 support

### DISCUSSION THREAD

**Discussion by sathyarajv** [danny-avila/LibreChat#2489] [Feature Requests & Suggestions]
OpenAI updated the Assistants API to V2 with changes to the tools and files. Refer the Migration steps [here](https://platform.openai.com/docs/assistants/migration/what-has-changed)

![image](https://github.com/danny-avila/LibreChat/assets/9957258/e3717564-24a7-443e-99ea-2cb8565028dd)

Known issues: The attached files of an existing Assistants are not loading in the UI but its available on the OpenAI platform.  Migrating to V2 will resolve this issue.

Any plan on this migration?

**Comment by danny-avila** [comment]
Absolutely planned, there is a lot of good stuff here, more than meets the eye, but particularly when it concerns usage in a UI.

**Reply by danny-avila** [reply]
See open issue I just made, thanks!

https://github.com/danny-avila/LibreChat/issues/2490
